test/provisioner/provision_azure.properties

```diff
@@ -0,0 +1,11 @@
+AZURE_SUBSCRIPTION_ID=${AZURE_SUBSCRIPTION_ID}
+# use az ad sp create-for-rbac   --role Contributor   --scopes "/subscriptions/$SUBSCRIPTION_ID" --query "{ client_id: appId, client_secret: password, tenant_id: tenant }"
+AZURE_CLIENT_ID=${AZURE_CLIENT_ID}
+AZURE_CLIENT_SECRET=${AZURE_CLIENT_SECRET}
```

Comment:
not sure how those variables are sourced, but you might want to quote the secret, sometimes there are characters like whitespace in them.